### PR TITLE
add storage profile detail

### PR DESCRIPTION
**Pull request by cangfengzhs**

### CODE REVIEW

File: src/common/interface/storage.thrift [hunk 1]
```diff
@@ -35,6 +35,7 @@ struct ResponseCommon {
     1: required list<PartitionResult>   failed_parts,
     // Query latency from storage service
     2: required i32                     latency_in_us,
+    3: optional map<string,i32>         latency_detail_us,
```
Comment (critical27) [inline src/common/interface/storage.thrift:38]: Better unify with code in storage, which uses int64_t

Comment (cangfengzhs) [reply src/common/interface/storage.thrift:38]: "latency_in_us" is int32. Do they have to be the same?Ignore "latency_in_us" or change "latency_in_us" to int64?

Comment (critical27) [reply src/common/interface/storage.thrift:38]: Both int32_t and int64_t is ok, just unify code in thrift and storage will be fine.

**Review by critical27 — APPROVED** [review]

File: src/common/interface/storage.thrift (hunk 1 above)
Comment (cangfengzhs) [reply src/common/interface/storage.thrift:38]: Ok, profile_detail_ map in storage has been changed.

**Comment by cangfengzhs** [comment]
Has been merged into nebula repo.

**Pull request closed without merging**